packages/salesforcedx-vscode-lwc/src/testSupport/workspace/getLwcTestRunnerExecutable.ts

```diff
@@ -38,23 +36,17 @@ export const getLwcTestRunnerExecutable = async (cwd: string) => {
     });
     if (lwcTestRunnerExecutable) {
       try {
-        const uri = vscode.Uri.file(lwcTestRunnerExecutable);
-        const stat = await vscode.workspace.fs.stat(uri);
-        if (stat) {
-          return lwcTestRunnerExecutable;
-        }
+        await stat(lwcTestRunnerExecutable);
```

Comment:
while we're here, look through this code to see if there's any way to make it less duplicative.

ex: can we 
1. get the executable (using a conditional for its path)
2. throw if not present
3. return

and maybe there's at the beginning to check for `lwc_test_no_lwc_testrunner_found` ?